Database/Updates/World/05-LoadInititalClothingTableDIDs.sql

```diff
@@ -0,0 +1,72 @@
+# Turban
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('135', '7', '268435488'); 
+# Jerkin
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('124', '7', '268435485'); 
+# Loose Shirt
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('2587', '7', '268435706'); 
+# Puffy Shirt
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('2591', '7', '268435710'); 
+# Puffy Tunic
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('2592', '7', '268435712'); 
+# Baggy Breeches
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('2603', '7', '268435704'); 
+# Baggy Pants
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('2598', '7', '268435699'); 
+# Pantaloons
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('2600', '7', '268435701'); 
+# Slippers
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('133', '7', '268435487'); 
+# Kasa
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('5901', '7', '268435865'); 
+# Flared Shirt
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('2588', '7', '268435707'); 
+# Flared Tunic
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('2594', '7', '268435714'); 
+# Loose Tunic
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('2593', '7', '268435713'); 
+# Flared Pants
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('2597', '7', '268435698'); 
+# Loose Breeches
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('2602', '7', '268435703'); 
+# Loose Pants
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('2601', '7', '268435702'); 
+# Beret
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('31236', '7', '268436908'); 
+# Poet's Shirt
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('31239', '7', '268436909'); 
+# Viamontian Leggings
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('31238', '7', '268436910'); 
+# Viamontian Laced Boots
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('31249', '7', '268436911'); 
+# Loafers
+INSERT INTO `ace_object_properties_did` (`aceObjectId`, `didPropertyId`, `propertyValue`) VALUES ('31237', '7', '268436912'); 
```

Comment:
As I put in my notes, this does belong there and could be moved a little later.   You can't test the work I did without this.   Again, you don't consider what the free developer is trying to do or say.   This is open source.   I think the word is collaboration.